docs/user-guide/cross-sections.ipynb

```diff
@@ -7,273 +7,203 @@
    "source": [
```

Comment:
<p>These two code cells, especially the helper function one, takes valuable space, which could serve better to showcase at least the "Cross-Section Method" section below that has important info (e.g. return type). </p><p> </p><p>What about moving these code cells to just after that section, i.e. just before Arbitrary Great Circle Arc (GCA), providing some info about the helper function is nothing related to cross-section itself?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I like that idea